extra/2-piping.js

```diff
@@ -16,26 +16,28 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  return a + b;
 }
 
-function multiply() {
-
+function multiply(a, b) {
+  return a * b;
 }
 
-function format() {
-
+function format(value) {
+  return "£" + value;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+let badCode = format(multiply(add(startingValue, 10), 2));
+
+/*  in this code there is space and no semicolons ; */
 
 /* BETTER PRACTICE */
 
-let goodCode =
+let goodCode = format(multiply(add(startingValue, 10), 2));
```

Comment:
It might be worth thinking about how you can make this easier to read.
Maybe we can do each part on a different line, and assign the result to a variable?